share/spice/movie/movie.js

```diff
@@ -90,37 +81,99 @@
                 }
             },
             relevancy: {
-                skip_words: ['movie', 'info', 'film', 'rt', 'rotten', 'tomatoes', 'rating', 'ratings', 'rotten'],
-                primary: [{
-                    key: 'title'
-                }]
+                skip_words: ['movie', 'info', 'film', 'rt', 'rotten', 'tomatoes', 'rating', 'ratings', 'rotten', 'cast', 'of'],
+                //primary: [{ // TODO: Need to fix this
+                //    key: 'title'
+                //}]
             },
-            onItemShown: function(item) {
-                if (!item.alternate_ids || !item.alternate_ids.imdb) { return; }
+            onItemShown: function (item) {
+                isCastQuery ? castPicture(item) : moviePicture(item);
+            }
+        });
+        function normalizeMovie(item) {
+            if (item.alternate_ids && item.alternate_ids.imdb) {
+                return {
+                    rating: Math.max(item.ratings.critics_score / 20, 0),
+                    icon_image: getImage(item.ratings.critics_rating),
+                    abstract: Handlebars.helpers.ellipsis(item.synopsis || item.critics_consensus, 200),
+                    heading: item.title,
+                    fallback_image: item.posters.detailed,
+                    image: null,
+                    url: item.links.alternate,
+                    is_retina: (DDG.is3x || DDG.is2x) ? "is_retina" : "no_retina"
+                };
+            }
+        }
+
+        function normalizeCast(item) {
+            return {
+                name: item.name,
+                character: item.characters && item.characters.join(", "),
+                heading: item.movie,
```

Comment:
We don't need to repeat the movie name in each title. We should emphasize the actors name in the detail pane and highlight their character's name